app/components/record/marc_contents_summary_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Record
+  class MarcContentsSummaryComponent < ViewComponent::Base
+    def initialize(document:)
+      super
+      @document = document
+    end
+
+    attr_reader :document
+
+    def render?
+      summaries.present?
```

Comment:
The partial rendered things besides `summary_struct`, didn't it?

Reply:
Ooops. Yup.